src/style.css

```diff
@@ -0,0 +1,121 @@
+* {
+  padding: 0;
+  box-sizing: border-box;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir.ttc) format("truetype");
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Heavy.ttf) format("truetype");
+  font-style: normal;
+  font-weight: 600;
+}
+
+body {
+  font-family: "Avenir", sans-serif;
+  font-size: 16px;
+}
+
+.search-form {
+  display: flex;
+  justify-content: center;
+  margin: 20px auto 0;
+}
+
+.search-form_big {
+  width: 100%;
+  height: 70px;
+}
+
+.search-form_small {
+  width: 100%;
+  height: 42px;
+}
+
+.search-form_label {
+  position: relative;
+  display: block;
+  width: 100%;
+}
+
+.search-form_label:after {
+  position: absolute;
+  content: "";
+  display: block;
+}
+
+.search-form_label_big:after {
+  top: 25px;
+  left: 26px;
+  width: 19px;
+  height: 19px;
+  background-image: url("images/Search.svg");
+  background-repeat: no-repeat;
+  background-size: contain;
+}
+
+.search-form_label_small:after {
+  top: 15px;
+  left: 13px;
+  width: 11px;
+  height: 11px;
+  background-image: url("images/Search.svg");
+  background-repeat: no-repeat;
+  background-size: contain;
+}
+
+.input {
+  font-family: inherit;
+  width: 100%;
+  height: 100%;
+  font-weight: 300;
+  color: #3d4e61;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  outline: none;
+}
+
+.input_big {
+  padding-left: 62px;
+  font-size: 16px;
+  line-height: 22px;
+  height: 70px;
+}
+
+.input_small {
+  padding-left: 33px;
+  font-size: 14px;
+  line-height: 19px;
+}
+
+.input:hover {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+}
+
+.input:focus {
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  font-weight: 600;
+  border-radius: 3px;
+}
+
+input::placeholder {
```

Comment:
don't style tags

Reply:
after fixing the edits does not work correctly